spec/unit/installer/xcode/multi_pods_project_generator_spec.rb

```diff
@@ -646,88 +646,123 @@ class Xcode
               banana_development_pods = project_by_target_map[banana_project].select { |pod_target| @generator.sandbox.local?(pod_target.pod_name) }
               @generator.share_development_pod_schemes(banana_project, banana_development_pods)
             end
-          end
 
-          it 'shares test schemes' do
-            @generator.installation_options.
-                stubs(:share_schemes_for_development_pods).
-                returns(true)
-            @generator.sandbox.stubs(:development_pods).returns('CoconutLib' => fixture('CoconutLib'))
+            it 'shares test schemes' do
```

Comment:
these tests were not indented properly. They are no nested `describe '#share_development_pod_schemes' do` correctly.